Assets/Fungus/Scripts/Editor/FlowchartWindow.cs

```diff
@@ -8,11 +8,94 @@
 using System.Linq;
 using System.Collections.Generic;
 using System.Reflection;
+using Object = UnityEngine.Object;
 
 namespace Fungus.EditorUtils
 {
     public class FlowchartWindow : EditorWindow
     {
+        protected class ClipboardObject
+        {
+            internal SerializedObject serializedObject;
+            internal Type type;
+
+            internal ClipboardObject(Object obj)
+            {
+                serializedObject = new SerializedObject(obj);
+                type = obj.GetType();
+            }
+        }
+
+        protected class BlockCopy
+        {
+            private SerializedObject block = null;
+            private List<ClipboardObject> commands = new List<ClipboardObject>();
+            private ClipboardObject eventHandler = null;
+
+            internal BlockCopy(Block block)
```

Comment:
Alright, I mean I'm by no means a C# guru so if something seems bizarre please let me know! I appreciate it. But I also hope you don't have to spend tons of time reviewing all my commits 😄 .